setup.py

```diff
@@ -23,13 +23,16 @@
         'symmetrical encryption of cached values using the python cryptography library'
     ),
     url='https://github.com/foundertherapy/django-redis-secure/',
-    download_url='https://github.com/foundertherapy/django-redis-secure/archive/1.0.0.tar.gz',
+    download_url='https://github.com/foundertherapy/django-redis-secure/archive/1.0.1.tar.gz',
     author='Ghaleb Khaled',
     author_email='ghaleb@foundertherapy.co',
     install_requires=[
-        'Django>=1.9.6',
+        'Django==1.9.6',
```

Comment:
When I was installing `secure redis` library using `pip install ... --upgrade` this was causing django to update to 1.10 which was conflicting with 53 project. But I think we should make it as you suggested `>= 1.9.0`